### ISSUE TITLE
Ganga doesn't handle spaces

### ISSUE THREAD

**Issue by mesmith75** [ganga-devs/ganga#859]
Do we want Ganga to be able to deal with spaces in the gangadir path?

The nightlies have been failing for a while at the first test because of this problem.
```python
Standard Error

#x1B[0;31mERROR    unknown options [defaults_DiracProxy]group#x1B[0;0m
#x1B[0;31mERROR    Error submitting job to Dirac: {'BadFile': ['/var/lib/jenkins/workspace/Nightly/python/gangadir', 'testing/simple_job/workspace/testframework/LocalXML/0/input/_input_sandbox_0.tgz'], 'TotalSize': 4953, 'Message': 'Input Sandbox is not valid', 'OK': False}#x1B[0;0m
#x1B[0;31mERROR
```

**Comment by rob-c** [comment]
This was fixed in 6.3 i decided not to spend energy back porting the fix as it was made to use the GangaDiracError.

**Comment by mesmith75** [comment]
@rob-c ah ok thanks, I'll close this then.
